modules/eks/echo-server/main.tf

```diff
@@ -50,8 +59,25 @@ module "echo_server" {
       name  = "ingress.alb.enabled"
       value = local.ingress_alb_enabled
       type  = "auto"
+    },
+    {
+      name  = "ingress.alb.access_logs.enabled"
+      value = local.alb_access_logs_enabled
+      type  = "auto"
+    },
+    {
+      name  = "ingress.alb.access_logs.s3_bucket_name"
+      value = var.alb_access_logs_s3_bucket_name == null ? "" : var.alb_access_logs_s3_bucket_name
+      type  = "auto"
+    },
+    {
+      name  = "ingress.alb.access_logs.s3_bucket_prefix"
+      value = var.alb_access_logs_s3_bucket_prefix
+      type  = "auto"
     }
-  ]
+    ],
```

Comment:
The formatting seems a little off here. did `terraform fmt` run ?

Reply:
Yeah I ran it that's how it formats it...